### ISSUE TITLE
Consider increasing the amount of available items across most generators

### ISSUE THREAD

**Issue by onetruepurple** [TerryCavanagh/diceydungeons.com#1399]
One of the more common complaints seems to be little variety in items available from run to run. I can't say I've ever been too bothered by it, but I see where they're coming from.

The game has probably enough equipment now that most chest/shop pools could be doubled without much trouble.

E.g. suppose that in episodes 2-6, Robot would be equally likely to encounter Cheat Code, Small Shield, Short Circuit, etc. as he would be likely to encounter Cauldron, Short Sword, Battle Axe, First Aid Kit, Dodge, Iron Armor, Magic Key, Rat, Target Shield, Bumpblade, Slime Ball, Broadsword, Crossbow, Catastrophe, Hookshot, Crowbar, Whisper, ...

Obviously this would not be an universally applicable change, as Thief and Witch avoid certain items thematically, and Jester is a special case. But even then it would be an improvement to make every run slightly more varied.

This could be additional incentive for checking out shops (rare drops) and Val (another semi-frequent complaint is that the trades are very situational).

**Comment by onetruepurple** [comment]
>  Thief and Witch avoid certain items thematically

Also worth considering if this aspect still makes sense, as it feels like a holdover from the early alphas a little (i.e. Warrior finding shields, but no armor or dodge, and vice versa for the Thief)

**Comment by TerryCavanagh** [comment]
Have been thinking about this issue a lot since launch, and have a lot to say about it. Will try to get my thoughts together on it soon.

Short answer - yes, but

**Comment by TerryCavanagh** [comment]
Ok: long answer.

Looking back at launch more than a year on, probably this is one of the major missteps I made. I think Dicey Dungeons could probably have benefited from further rounds of playtesting with more people, and if I'd worked on the the launch version for another three months, say, this is likely one of the major changes I have ended up making (along with changes to the meta game structure, some changes to Witch, and likely cutting about 12 episodes).

At this point, though, I'm finishing up ports and firming up the game, and starting to think about DLC and new projects beyond that, and I'm not planning to make as major a core change to the game as this.

I've learned a lot making Dicey Dungeons, and am really excited about doing more work in this space. Something to keep in mind for whatever's next!
